### PR TITLE
Support both es6 and es7

### PR DESCRIPTION
**Pull request by lferran**

### CODE REVIEW

**Review by masipcat — APPROVED** [review]
Looks good!

**Review by masipcat — APPROVED** [review]
Double approve!

File: guillotina_elasticsearch/__init__.py [hunk 1]
```diff
@@ -4,6 +4,13 @@
 from guillotina.component import get_utilities_for
 from guillotina.content import IResourceFactory
 from guillotina.utils import get_dotted_name
+from packaging import version
+
+import aioelasticsearch
+
+
+ES_CLIENT_VERSION = version.parse(aioelasticsearch.__version__)
+ELASTIC6 = ES_CLIENT_VERSION.minor == 5
```
Comment (vangheem) [inline guillotina_elasticsearch/__init__.py:13]: nice!

**Review by vangheem — APPROVED** [review]
nicely done!

**Comment by lferran** [comment]
can you guys squash and merge? I'm not allowed for some reason

**Comment by masipcat** [comment]
done! I'll do a release

**Comment by masipcat** [comment]
released!

**Pull request merged**